Sources/PaylikeSDK/Views/Components/CardValidationCodeField.swift

```diff
@@ -10,16 +10,15 @@ import SwiftUI
 struct CardValidationCodeField: View {
     @Binding public var cvc: String
     public var isValid: Bool
-    @State public var isEditing: Bool = false
```

Comment:
Why is this deleted?

Reply:
onEditingChanged is not a SecureField property, so it cannot work the same way as textField
An alternative might be to use onSubmit or onCommit: https://developer.apple.com/documentation/swiftui/view/onsubmit(of:_:)